common/spec/dependabot/clients/azure_spec.rb

```diff
@@ -181,6 +181,39 @@
     end
   end
 
+  describe "#create_pull_request" do
+    subject do
+      client.create_pull_request("pr_name", "source_branch", "target_branch",
+                                 "", [], nil)
+    end
+
+    let(:pull_request_url) { repo_url + "/pullrequests?api-version=5.0" }
+
+    context "when response is 403 & tags creation is forbidden" do
+      before do
+        stub_request(:post, pull_request_url).
+          with(basic_auth: [username, password]).
+          to_return(status: 403, body: JSON.unparse({ message: "TF401289: The current user does not have permissions to create tags" }))
```

Comment:
```suggestion
          to_return(
            status: 403,
            body: { message: "TF401289: The current user does not have permissions to create tags" }.to_json
          )
```

The linter is currently failing because this line is too long, and the `.to_json` is a bit more in line with the rest of the codebase.

Reply:
Sure will be addressing this.